python/servo/bootstrap.py

```diff
@@ -103,8 +110,20 @@ def linux(context, force=False):
                 'clang', 'clang-libs', 'autoconf213', 'python3-devel',
                 'gstreamer1-devel', 'gstreamer1-plugins-base-devel',
                 'gstreamer1-plugins-bad-free-devel']
-
-    installed_something = install_linux_deps(context, pkgs_apt, pkgs_dnf, force)
+    pkgs_xbps = ['libtool', 'gcc', 'libXi-devel', 'freetype-devel',
+                 'libunwind-devel', 'MesaLib-devel', 'glib-devel', 'pkg-config',
+                 'libX11-devel', 'libXrandr-devel', 'gperf', 'bzip2-devel',
+                 'fontconfig-devel', 'cabextract', 'expat-devel', 'cmake',
+                 'cmake', 'libXcursor-devel', 'libXmu-devel', 'dbus-devel',
+                 'ncurses-devel', 'harfbuzz-devel', 'ccache', 'glu-devel',
+                 'clang', 'gstreamer1-devel', 'autoconf213',
+                 'gst-plugins-base1-devel', 'gst-plugins-bad1-devel']
+    if context.distro == "Ubuntu" and context.distro_version != "14.04":
+        pkgs_apt += ['libgstreamer1.0-dev', 'libgstreamer-plugins-base1.0-dev',
+                     'libgstreamer-plugins-bad1.0-dev']
```

Comment:
Why is that change necessary?

Reply:
I think it was a mistakes happens during a conflict, I'll remove that soon.